app/Repositories/PostRepository.php

```diff
@@ -93,7 +113,13 @@ public function create(array $data, $signupId)
 
         // Edit the image if there is one
         if (isset($data['file'])) {
-            $editedImage = $this->crop($data, $post->id);
+            $this->crop($data, $post->id);
+        }
+
+        // Save the new post in Customer.io, via Blink.
+        if (config('features.blink')) {
```

Comment:
This type of logic would be perfect for the `[PostService`](https://github.com/DoSomething/rogue/blob/eb6d59a920611f5d022574660c0920182a55ebff/app/Services/PostService.php#L34) class which was meant to handle more business-type logic around creating posts so we don't conflate database concerns with service concerns. 

I know there was some concern about not needed all of this separation, but I think it makes sense now that we are adding other logic around creating posts instead of just storing in the DB.